cmake/O2Utils.cmake

```diff
@@ -280,14 +280,6 @@ macro(O2_GENERATE_LIBRARY)
   # Install all the public headers
   install(DIRECTORY include/${MODULE_NAME} DESTINATION include)
 
-  Set(LIBRARY_NAME)
-  Set(DICTIONARY)
```

Comment:
It does not make sense to delete the library name at the end of the macro, cleaning all the other variables might be useful however.